Containers/nextcloud/Dockerfile

```diff
@@ -105,7 +105,7 @@ RUN { \
 VOLUME /var/www/html
 
 
-ENV NEXTCLOUD_VERSION 23.0.1
+ENV NEXTCLOUD_VERSION 23.0.3
```

Comment:
I hope the mentioned fix in server will be reviewed, merged and backported for that release.